contrib/remote-dev/README.md

```diff
@@ -0,0 +1,165 @@
+# Remote Development
+
+## 1. Overview
+
+Remote Development can allow users to develop locally with PAI's resources and improve users’ development experience.
+
+Basicaly, the main process is to ask PAI for a container with resources, and use local editors or IDEs to develope and debug in this container using ssh.
+
+Support IDEs or editors:
+- CLI
+- Visual Studio Code
+- PyCharm
+
+Support OS:
+- Ubuntu 16.04
+- Ubuntu 18.04
+- Windows 10
+
+***NOTICE: This is still an experiment solution and may be changed in future release.***
+
+## 2. Directory Structure
+
+```
+remote-dev/
+├── README.md
+├── conf
+│   ├── clusters.template        # PAI cluster template conf
+│   └── job.template             # PAI template job
+└── start.py                     # start script
+```
+
+## 3. Dependencies
+
+- This sub project depend on [PAI's Python SDK](https://github.com/microsoft/pai/tree/master/contrib/python-sdk), please install and test the sdk first.
+
+- The `start.py` script requires python3, and we only tested it on `py3.5+` environment, and ruquired following moduls:
+    - requests
```

Comment:
write down the `pip install` command?

Reply:
These modules are installed by default. I will remove this module request.